### ISSUE TITLE
nxxm cl => pull access denied for nxxm/nxxm/ci/ubuntu-cpp

### ISSUE THREAD

**Issue by ArthurSonzogni** [nxxm/nxxm#6]
See https://gitlab.com/ArthurSonzogni/ftxui/pipelines/45848823/failures

I used
~~~bash
nxxm ci
~~~

Then gitlab respond with:
~~~
Running with gitlab-runner 11.7.0-rc1 (6e20bd76)
  on docker-auto-scale ed2dce3a
Using Docker executor with image registry.gitlab.com/nxxm/nxxm/ci/ubuntu-cpp:latest ...
Pulling docker image registry.gitlab.com/nxxm/nxxm/ci/ubuntu-cpp:latest ...
ERROR: Job failed: Error response from daemon: pull access denied for registry.gitlab.com/nxxm/nxxm/ci/ubuntu-cpp, repository does not exist or may require 'docker login' (executor_docker.go:168:0s)
~~~

**Comment by daminetreg** [comment]
Hi @ArthurSonzogni,

Thanks for using nxxm in FTXUI, which is an awesome project. I'm looking to add the command line html ui renderer support, but based on what you explained in the other issue it might already be there.

Please retry now it is fixed, the docker image is accessible to everyone registry.gitlab.com/nxxm/nxxm/ci/ubuntu-cpp:latest

Thanks for your testing it's invaluable.  :heart: :smile:

**Comment by ArthurSonzogni** [comment]
Thanks, it works: https://gitlab.com/ArthurSonzogni/ftxui/-/jobs/155261905
It worked, but failed at another step. So it is a different issue.

A command line html ui renderer support: wow! The web is hard. Good luck!
What will be the name of it?

On my side, I am doing the same, but with json. It will be a sort of UI to explore them.

**Comment by daminetreg** [comment]
I didn’t come up with a name I would like to contribute something like this to ftxui.

The issue seems to be that the nxxm we ship in the registry is for some reason not up-to-date. I’ll check why and come back here once it is fixed.

**Comment by daminetreg** [comment]
I fixed the issue. Indeed the default file generated by the ci command doesn't include the [nxxm/upgrd](https://github.com/nxxm/upgrd) lib option : `--force-upgrade`.

Therefore it uses always the one in the registry, which happened to not be totally the last version (it was a developer build of nxxm named as well 0.0.7 but not of the correct SHA-1 ^^), I uploaded the official release. I'll ensure this won't happen anymore in the future.

Next github releases of nxxm will always trigger docker registry regeneration.

Now your build has passed ( I triggered it ^^ ). Therefore I'll close this as it is solved.

See https://gitlab.com/ArthurSonzogni/ftxui/-/jobs/155320770
